os/fs/driver/mtd/smart.c

```diff
@@ -4887,8 +4887,34 @@ static int smart_ioctl(FAR struct inode *inode, int cmd, unsigned long arg)
 
 	case BIOC_BULKERASE:
 		ret = MTD_IOCTL(dev->mtd, MTDIOC_BULKERASE, 0);
+
+		lldbg("Format Finished\n");
```

Comment:
If you want to print this always, please use `printf` instead of `lldbg`.

Reply:
I know, application has another log. so it will be displayed. I will change this to fdbg instead of lldbg.